src/daemon-base/__CEPH_BASE_PACKAGES__

```diff
@@ -12,4 +12,7 @@
         gdisk \
         __RADOSGW_PACKAGE__ \
         __GANESHA_PACKAGES__ \
-        __ISCSI_PACKAGES__
+        __ISCSI_PACKAGES__ \
+        ceph-fuse \
```

Comment:
I'm wondering if it makes sense to put those packages under __CSI_PACKAGES__ (to be created) wdyt ?
Also I'm not sure those packages are available for all distro/release, did you checked that ?

Reply:
Yes, I thought about `CSI_PACKAGES`, I guess it makes sense to have it if packages are named differently in other distros, I haven't checked that yet.

Reply:
I've checked and the three packages are available for all distro/releases
I've updated the PR because there was a typo on `__CSI_PACKAGES__` (missing S) and also remove the other empty occurence in ceph-release directory